dockerbuild/Dockerfile

```diff
@@ -17,7 +16,13 @@ RUN apt-get -qq update && apt-get -qq dist-upgrade && \
   libsqlcipher-dev && \
   # git-lfs
   git lfs install && \
-  apt-get purge -y --auto-remove && rm -rf /var/lib/apt/lists/*
+  # used by flathub client \
+  apt-get -qq install -y --no-install-recommends flatpak ostree gettext appstream-util curl && \
+  apt-get -qq install -y python3 python3-dev python3-pip libgirepository1.0-dev gir1.2-ostree-1.0 libcairo2-dev && \
+  curl -sSL https://raw.githubusercontent.com/flatpak/flat-manager/master/flat-manager-client -o /usr/local/bin/flat-manager-client && \
+  chmod +x /usr/local/bin/flat-manager-client && \
+  pip3 install aiohttp tenacity PyGObject && \
+  apt-get purge -y --auto-remove && rm -rf /var/lib/apt/lists/
```

Comment:
Is there not a preexisting docker image for building flatpaks? Looks like the steps are separate, so it doesn't need  to be the same docker image that runs electron build.